pyo3-ffi/src/compat.rs

```diff
@@ -42,3 +44,16 @@ pub unsafe fn PyDict_GetItemRef(
         -1
     }
 }
+
+#[cfg_attr(docsrs, doc(cfg(all)))]
```

Comment:
Ah, I see what might have gone awry in the `PyDict_GetItemRef` PR - `all()` is supposed to be invoked as a function. Try this:

```suggestion
#[cfg_attr(docsrs, doc(cfg(all())))]
```

Then that "available on **all** only" bit might be removed from the docs. 🤔 

If it works, maybe adjust the `PyDict_GetItemRef` bindings above too?

Reply:
Nice! That did fix the weird rendering on the nightly docs build.